json-examples/svc/provider-network-interas-option-a.json

```diff
@@ -0,0 +1,51 @@
+{
+   "attachment-circuits":{
+      "ac":{
+         "name":"IPT-CUST-ABC",
+         "customer-name":"CUST-ABC",
+         "description":"CUST-ABC-113",
+         "peer-sap-id":"sap#113",
+         "ip-connection":{
+            "ipv4":{
+               "local-address":"113.1.1.1",
+               "prefix-length":31,
+               "address-allocation-type":"ac-common:static-address"
+            }
+         },
+         "l2-connection":{
+            "encapsulation":{
+               "dot1q":{
+                  "tag-type":"vpn-common:c-vlan",
+                  "cvlan-id":"113"
+               }
+            },
+            "bearer-reference":"line-113"
+         },
+         "routing-protocols":{
+            "routing-protocol":{
+               "id":"BGP-Single-Access",
+               "type":"vpn-common:bgp-routing",
+               "bgp":{
+                  "peer-groups":{
+                     "peer-group":{
+                        "name":"IPT-CUST-ABC",
+                        "peer-as":1234,
```

Comment:
```suggestion
                        "peer-as":65536,
```

Reply:
Use ASN reserved for documentation defined in https://datatracker.ietf.org/doc/rfc5398/